src/common/components/ui/formik/types.ts

```diff
@@ -31,7 +31,17 @@ export interface SelectFieldProps extends FieldProps {
   // onBlur?: (event: React.FormEvent<HTMLSelectElement>) => void;
 }
 
+export interface MultiSelectFieldProps extends FieldProps {
+  options: (SelectOptionProps & { displayName?: string })[];
```

Comment:
lets require `id` prop. 

```suggestion
  options: (SelectOptionProps & { id: string; displayName?: string })[];
```

it will make our life easier in `MultiSelectField`
```
<SelectOption
                key={option.id}
                id={option.id}
                value={option.value}
                // itemCount={itemCount} TODO: This is broken in Patternfly ATM
              >
                {option.displayName || option.value}
              </SelectOption>
```

Reply:
done